app/views/miq_ae_class/_method_form.html.haml

```diff
@@ -47,7 +47,7 @@
         .col-md-8
           = h(format_timezone(@ae_method.created_on, Time.zone, "gtl"))
   %hr
-  %h3= _("Data")
+  %h3= (@edit[:new][:location] != 'builtin') ? _("Data") : _('Builtin name')
```

Comment:
@mfalesni why not switch this condition like the one  app/views/miq_ae_class/_method_inputs.html.haml
 ```
 %h3= (@edit[:new][:location] == 'builtin') ? _('Builtin name') : _("Data")
```